voltdbclient/response.go

```diff
@@ -303,7 +303,8 @@ func deserializeTableForResult(r io.Reader) (rowsAff int64, err error) {
 	if err != nil {
 		return 0, err
 	}
-	if strings.Compare("modified_tuples", cname) != 0 {
+
+	if strings.Compare("modified_tuples", cname) != 0 && cname != "STATUS" {
```

Comment:
The original code used `strings.Compare()`, which is less preferred. It would be nice to be consistent on using `!=` or `==` if we decide to keep this check.